README.md

```diff
@@ -27,13 +30,15 @@ on:
   discussion_comment:
     types:
       - created
+  permissions:
```

Comment:
Should be defined under [`jobs.\<jobid\>.permissions`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idpermissions)